assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/types.ts

```diff
@@ -1,5 +1,6 @@
-import { Entity, EntityId, Price, Ticket } from '../../../services/apollo/types';
+import { Price, Ticket } from '../../../services/apollo/types';
```

Comment:
these entities won't be shared with other parts of the app?
this is why they live in `eventEditor` services, right?

Reply:
Yes, they are specific to EDTR.